src/pybind/mgr/dashboard/tests/test_rbd_mirroring.py

```diff
@@ -288,7 +288,7 @@ def test_summary(self, mock_rbd, has_perms_mock):
         self.assertStatus(200)
 
         summary = self.json_body()['rbd_mirroring']
-        self.assertEqual(summary, {'errors': 0, 'warnings': 2})
+        self.assertEqual(summary, {'errors': 0, 'warnings': 0})
```

Comment:
Can we try to generate warnings or error just for testing purposes?

Reply:
we'll need to mock the callback for that i guess. let me check it.

Reply:
done